wagtail/wagtailimages/utils.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import, unicode_literals
+
+
+# Helper functions for migrating the Rendition.filter foreign key to the filter_spec field,
+# and the corresponding reverse migration
+
+def get_fill_filter_spec_migrations(app_name, rendition_model_name):
+
+    def fill_filter_spec_forward(apps, schema_editor):
+        # Populate Rendition.filter_spec with the spec string of the corresponding Filter object
+        Rendition = apps.get_model(app_name, rendition_model_name)
+        db_alias = schema_editor.connection.alias
```

Comment:
That's how it is done in the documentation, I just copied it from there: https://docs.djangoproject.com/en/1.10/ref/migration-operations/#runpython

And specifically the warning:

```
RunPython does not magically alter the connection of the models for you; any model methods you call will go to the default database unless you give them the current database alias (available from schema_editor.connection.alias, where schema_editor is the second argument to your function).
```